llama_stack/distribution/providers.py

```diff
@@ -57,3 +66,63 @@ async def inspect_provider(self, provider_id: str) -> ProviderInfo:
                 return p
 
         raise ValueError(f"Provider {provider_id} not found")
+
+    async def get_providers_health(self) -> Dict[str, Dict[str, HealthResponse]]:
+        """Get health status for all providers.
+
+        Returns:
+            Dict[str, Dict[str, HealthResponse]]: A dictionary mapping API names to provider health statuses.
+                Each API maps to a dictionary of provider IDs to their health responses.
+        """
+        providers_health: Dict[str, Dict[str, HealthResponse]] = {}
+        timeout = 1.0
+
+        async def check_provider_health(impl: Any) -> tuple[str, HealthResponse]:
+            """Check health of a single provider.
```

Comment:
this documentation is quite unnecessary, the naming is quite enough

Reply:
So do you want to remove the entire docstring?